configure.ac

```diff
@@ -106,17 +106,10 @@ AC_ARG_ENABLE([upnp-default],
   [use_upnp_default=$enableval],
   [use_upnp_default=no])
 
-zusetests=no
-case $host in
-  *linux*)
-     zusetests=yes
-  ;;
-esac
-
 AC_ARG_ENABLE(tests,
-    AS_HELP_STRING([--disable-tests],[do not compile tests (default is to compile)]),
-    [use_tests=$enableval],
-    [use_tests=$zusetests])
+    AS_HELP_STRING([--enable-tests],[compile tests (default is not to compile)]),
+    [use_tests=$withval],
```

Comment:
Seem like `$withval` should be using with `--with-xxx` instead. For `--enable-xxx` it look like we need to use `$enableval` instead.